LICENSES/CC-PDDC.txt

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Is this license compatible with `LGPLv3`?

Reply:
i hope so :D there was some public domain license stuff in sdb!

Reply:
The original files just had "public domain" (also checked the original cdb.c), which I hope is the same as https://spdx.org/licenses/CC-PDDC.html . I think if it's public domain you can do whatever you want it it.